File: _examples/python/tuples.py
# 创建元组
person = (25, "Alice", 1.68)
coords = 10, 20  # 括号可省略

# 访问元素
age = person[0]
name = person[1]

# 解包
age, name, height = person

# 命名元组
from collections import namedtuple
Point = namedtuple('Point', ['x', 'y'])
p = Point(10, 20)

File: _examples/typescript/tuples.ts
// TypeScript 元组类型
const person: [number, string, number] = [25, "Alice", 1.68];

// 解构
const [age, name, height] = person;

// 命名元组 (TS 4.0+)
type Person = [age: number, name: string, height: number];

// 只读元组
const point: readonly [number, number] = [10, 20];